.github/workflows/jobs-coral.yaml

```
@@ -1,15 +1,33 @@
-name: Coral PR Pipeline
+# This workflow is not automatically triggered, but needs to be
+# called from e.g. another workflow file. It enables us to
+# reuse the jobs run here in different workflows.
+# Please note that in order to run this jobs, the
+# code needs to be checked already out when using the job
+name: Coral related jobs
 
 on:
-  pull_request:
-    paths:
-      - 'coral/**'
-      - '!coral/types/api.d.ts'
+  workflow_call
 
 permissions:
   contents: read
 
 jobs:
+  type-check:
+    name: TypeScript compiler
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # v3.5.0
+        with:
+          persist-credentials: false
+
+      - name: Setup coral
+        uses: ./.github/actions/setup-coral
```

should this point to the file instead of the directory ? 🤔 
```suggestion
        uses: ./.github/actions/setup-coral/action.yaml
```

---

@DocEmmetBrown  based on the github docs, a composite action has to have the name "action.yaml" and then is referred to only by directory (see https://docs.github.com/en/actions/creating-actions/creating-a-composite-action)

---

@programmiri : TIL ! Thanks a lot !